[JIT] Rename canonicalize ops

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #38746 [JIT] Factor out aliases to separate test
* #38735 [JIT] Normalize op aliases
* **#38734 [JIT] Rename canonicalize ops**

As far as I can tell, this pass only exists to canonicalize ops that are generating in the graph fuser, so it's kind of a misnomer.

Differential Revision: [D21673109](https://our.internmc.facebook.com/intern/diff/D21673109)

Commit:
[JIT] Rename canonicalize ops

[ghstack-poisoned]

jamesr66a reviewed (approved):
![image](https://user-images.githubusercontent.com/4685384/82371656-68938600-99cf-11ea-95ae-7afa0e59eb55.png)

Commit:
Update on "[JIT] Rename canonicalize ops"


As far as I can tell, this pass only exists to canonicalize ops that are generating in the graph fuser, so it's kind of a misnomer.

[ghstack-poisoned]

Commit:
Update on "[JIT] Rename canonicalize ops"


As far as I can tell, this pass only exists to canonicalize ops that are generating in the graph fuser, so it's kind of a misnomer.

[ghstack-poisoned]
